### DISCUSSION TITLE
Problems with links to resources in the "static" folder (or locally as a "page bundle")

### DISCUSSION THREAD

**Discussion by vitabef** [gethinode/hinode#1113] [Help] [answered]
I want to place links on my site for downloading .zip and .pdf files. Unfortunately, with the new link functionality, this doesn't work because it checks whether the page exists. 
For example:
`[Download as ZIP](/files/MyFile.zip "Download MyFile.zip")`
In this case, MyFile.zip is physically located under /static/files/
`**ERROR partial [assets/link.html] - Cannot find page: /files/MyFile.zip**`

**Comment by markdumay** [comment]
Got it, I did not think of this use case yet... I've prepared a new branch `download-link`, care to test it?

**Reply by vitabef** [reply]
Thank you for the super quick implementation! I encountered this error message while testing:
`execute of template failed: template: partials/assets/link.html:77:24: executing "partials/assets/link.html" at <not>: wrong number of args for not: want 1 got 0`
I think it's this line:
`...
77: {{- if and $ref not $error -}}
...`

in /layouts/partials/assets/link.html

**Comment by vitabef** [comment]
`{{- if and $ref (not $error) -}}`

**Reply by markdumay** [reply, marked as answer]
Thanks, I added it to the branch. I forgot to run the code against the site example before pushing it to GitHub.
